tests/data/pipeline_before_feb_fields.geojson

```diff
@@ -0,0 +1,50 @@
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "type": "Feature",
+      "properties": {
+        "scheme_reference": "bb649a68-5fc0-423a-b679-25dc43a8d020",
+        "intervention_type": "other",
+        "pipeline": {
```

Comment:
Note lack of new fields. This is produced with the current live pipeline version